src/rebuild/srv.c

```diff
@@ -1436,8 +1422,8 @@ rebuild_task_complete_schedule(struct rebuild_task *task, struct ds_pool *pool,
 
 		if (task->dst_rebuild_op == RB_OP_RECLAIM ||
 		    task->dst_rebuild_op == RB_OP_FAIL_RECLAIM) {
-			DL_INFO(ret, DF_UUID" retry opc %u/%u", DP_UUID(task->dst_pool_uuid),
-				task->dst_rebuild_op, task->dst_map_ver);
+			DL_INFO(ret, DF_RB " retry, dst_map_ver %u", DP_RB_RGT(rgt),
```

Comment:
here the "task->dst_rebuild_op" possibly be different with rgt->rgt_opc.
For example a REBUILD failed and here schedule a FAIL_RECLAIM task.
So better to log the retry opc same as original code, to avoid confuse.
same for several other places in this function.

Reply:
In this function I'll restore the logging to have the task->dst_rebuild_op and remove DF_RB.

Maybe in a future patch, if it is functionally correct, I could use a format like this to get part of DF_RB information while keeping the task information too:
DF_UUID "%u/%u opc %u/%u ...",
DP_UUID(), rgt->rgt_rebuild_ver, rgt->rgt_rebuild_gen, task->dst_rebuild_op, task->dst_rebuild_gen

Doing that can make it easier to find these slightly different log messages with the other logs that use DF_RB